rollup.config.js

```diff
@@ -11,7 +11,8 @@ const glob = require('glob');
 const path = require('path');
 const { uglify } = require('rollup-plugin-uglify');
 
-const base = path.resolve(process.cwd(), 'src');
+// use '/' dir seps on win32, to satisfy sass-loader/LibSass; otherwise crash
```

Comment:
I suggest to sharpen the comment, like this:
// map platform native backslash on win32 to platform neutral forward slash (which path resolve needs)

Reply:
ok, docu updated!
(the forward slash is needed by sass-loader or LibSass, not by path.resolve)